.circleci/regenerate.py

```diff
@@ -30,7 +30,7 @@ def build_workflows(prefix='', filter_branch=None, upload=False, indentation=6,
     for btype in ["wheel", "conda"]:
         for os_type in ["linux", "macos", "win"]:
             python_versions = PYTHON_VERSIONS
-            cu_versions_dict = {"linux": ["cpu", "cu102", "cu111", "rocm4.0.1", "rocm4.1"],
+            cu_versions_dict = {"linux": ["cpu", "cu102", "cu111", "rocm4.0.1", "rocm4.1", "rocm4.2"],
```

Comment:
Why do we need to keep 3 version of rocm here? Can we limit it down to just 2? (4.1 and 4.2 for example)

Reply:
We at least need N and N-1.  I thought we were following the established tradition of keeping latest and two prior rocm versions.  Is this not the same for torchvision?  Let us know.  Like I said, keeping latest and one prior is useful.

Reply:
Done.  Removed rocm4.0.1 build in commit 5098b4331e2c7f929288517aea04629d513d8b82.